adam-casey-devetry/showmechange-FE
# Make duplicate of project

**This is what I did** : 

- Burned everything to the ground and started all over again 

**To test** : 

- Run `npm start `

- React sever should start and home page should state 'Show me change'
